firmware/klipper_configurations/SKR_1.3/Voron2_SKR_13_Config.cfg

```diff
@@ -575,9 +577,9 @@ gcode:
     
 ## 	Thermistor Types
```

Comment:
Since you added the link to the klipper docs in the comment in the extruder section you can delete the whole "Thermistor Types" comment block in all the configs.